blueprints/util-test/qunit-rfc-232-files/tests/unit/utils/__name__-test.js

```diff
@@ -0,0 +1,13 @@
+import <%= camelizedModuleName %> from '<%= dasherizedModulePrefix %>/utils/<%= dasherizedModuleName %>';
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('<%= friendlyTestName %>', function(hooks) {
+  setupTest(hooks);
```

Comment:
Hmm. In general I think that we probably don’t need this? I would expect utils to invoke the util method and test its inputs and outputs. Let’s remove `setupTest` here for now?

Reply:
agreed 👍